app/grandchallenge/challenges/views.py

```diff
@@ -134,23 +137,25 @@ def get_form_kwargs(self):
         return kwargs
 
     def get_success_url(self):
-        return reverse("challenges:list")
+        return reverse("challenges:requests-list")
 
 
-class ChallengeRequestList(
-    LoginRequiredMixin, PermissionRequiredMixin, ListView
-):
+class ChallengeRequestList(LoginRequiredMixin, PermissionListMixin, ListView):
     model = ChallengeRequest
-    permission_required = "challenges.view_challengerequest"
+    permission_required = "view_challengerequest"
+    raise_exception = True
+    login_url = reverse_lazy("account_login")
 
 
 class ChallengeRequestDetail(
     LoginRequiredMixin,
-    PermissionRequiredMixin,
+    ObjectPermissionRequiredMixin,
```

Comment:
Please make sure that there are tests for all of these changes - that one request creator cannot access those of another, that the list view is filtered etc.